collection-scripts/gather_network_observability

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NO_NS="$(oc get subs -A -o template --template '{{range .items}}{{if eq .spec.name "netobserv-operator"}}{{.metadata.namespace}}{{end}}{{end}}')"
+
+if [ -z "${NO_NS}" ]; then
+    echo "INFO: Network Observability namespace is not detected. Skipping."
+    exit 0
+fi
+
+function get_netobserv_cluster_crs() {
+    CRD="flowcollectors.flows.netobserv.io"
+    oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" "${CRD}"
+    obj=$(oc get ${CRD} -o custom-columns=:metadata.name --no-headers)
+    FC_NS=$(oc get ${CRD} ${obj} --output=jsonpath='{.spec.namespace}')
```

Comment:
The scoping here is odd (I know this works in bash) but it makes code like this very hard to maintain and debug.